vendor/wp-now/src/download.ts

```diff
@@ -46,7 +46,6 @@ interface DownloadFileAndUnzipResult {
 	statusCode: number;
 }
 
-followRedirects.maxRedirects = 5;
```

Comment:
I also noticed this Typescript issue when checking the code. In theory, `maxRedirects` can be used so this seems an issue in the type definition. I opened the PR https://github.com/Automattic/studio/pull/427 to address it.